Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -56,8 +63,9 @@ function InternalInit
     $repositoryIdGuid = [Guid]::Parse($repositoryId);
     Write-Verbose "Fetching the pull request object with id $pullRequestId"
 
-    $script:pullRequest = [PsWorkarounds.Helper]::GetPullRequestObject($script:gitClient, $script:project, $repositoryIdGuid, $pullRequestId);
-     
+    $request = InvokeByReflection $script:gitClient "GetPullRequestAsync" @([String], [Guid], [int]) @($script:project.ToString(),  [Guid]::Parse($repositoryIdGuid), $pullRequestId)
```

Comment:
The old workaround doesn't seem to work when using a core clr agent, so I figured out a new one.